concent_api/conductor/tasks.py

```diff
@@ -54,11 +53,9 @@ def blender_verification_request(
         f'With blender_crop_script: {bool(blender_crop_script)}',
         subtask_id=subtask_id
     )
-    validate_frames(frames)
```

Comment:
I understand the purpose of this change, I want just to point out that doing this in core instead of here will not start whole verification process. Wouldn't this result in a different outcome for requestor/provider when this validation fails? @cameel ?

Reply:
I think it's a change for the better. If the list of frames is invalid, `core`, should immediately reject the message as invalid without even starting the verification.

Actually, even after this change the validation happens too late. It's now in `send_blender_verification_request()` but this function is called after we have already accepted the request, updated the subtask and committed the changes to the database. It should be performed along with all the other validations for `TaskToCompute`.

Also, conductor is a separate component so I think it should validate its input separately from `core`. So it would be fine to have this check here too. Or at least `assert` that the frames are valid - just like it already does for example for `output_format`.

Reply:
ok, I will move it insisde `validate_task_to_compute`